boards/mpfs-icicle-kit-es/def_config

```diff
@@ -1,4 +1,3 @@
-# Generated by Kconfiglib (https://github.com/ulfalizer/Kconfiglib)
```

Comment:
Do we need to make change  of this file since we add bfn_defcongih

Reply:
Reverted to 2021.08 tag